app/src/test/java/ca/klapstein/baudit/Data/PhoneNumberTest.java

```diff
@@ -4,34 +4,66 @@
 import org.junit.Before;
 import org.junit.Test;
 
+import static junit.framework.TestCase.fail;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 
 public class PhoneNumberTest {
     @Before
-    public void setUp() {
-    }
+    public void setUp() {}
 
     @After
-    public void tearDown() {
-    }
+    public void tearDown() {}
 
     @Test
-    public void test_PhoneNumberConstructor() {
-        PhoneNumber phoneNumber = new PhoneNumber("111-111-1111");
+    public void testPhoneNumberConstructor() {
+        PhoneNumber phoneNumber = new PhoneNumber("(780)4444444");
         assertNotNull(phoneNumber);
-        assertEquals(phoneNumber.getPhoneNumber(), "111-111-1111");
+        assertEquals("7804444444", phoneNumber.getPhoneNumber());
     }
 
     @Test
-    public void test_getPhoneNumber() {
+    public void testGetPhoneNumber() {
+        PhoneNumber phoneNumber = null;
+
+        try {
+            phoneNumber = new PhoneNumber("012-345-6789");
```

Comment:
using `fail()` is redundant.

If the test does throw an `IllegalArgumentException` it will fail. This will also give us better context as to why the test failed as it will give the entire stacktrace.

Using a `try` `catch` and calling `fail()` removes this context and the returning stacktrace, thus, making the test less effective.